app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder

```diff
@@ -2,9 +2,15 @@
 
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+json.id paymentable.houid
 
 json.object 'offline_transaction_charge'
 
+json.created paymentable.created.to_i
```

Comment:
Same question as above.

Reply:
This one is also in the subtransaction_payment jbuilder, but it's recording the datetime for the subtransaction_payment, not the paymentable. Won't that be a problem?
```ruby
json.created subtransaction_payment.created.to_i
```

Reply:
/ can we assume the time or creation will be the same?

Reply:
@wwahammy what are your thoughts on this?

Reply:
Let's use `json.created paymentable.created.to_i` and we should document the weirdness. Please file an issue to explain the weird behavior here.

Reply:
Done